src/dropdown/Dropdown.d.ts

```diff
@@ -0,0 +1,68 @@
+// TODO: use react-select types https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-select/index.d.ts
+
+import { CSSProperties, PureComponent, SyntheticEvent } from 'react'
+import * as t from 'tcomb';
+
+type Value = number | string | object | object[];
+
+type MenuItem = {
+  focusedOption: object,
+  instancePrefix: string,
+  onFocus: (e: SyntheticEvent<HTMLDivElement>) => void,
+  onSelect: (option: object, e: SyntheticEvent<HTMLDivElement>) => void,
+  groupByKey?: string,
+  optionClassName: string,
+  optionComponent: JSX.Element,
+  optionRenderer: () => JSX.Element,
+  optionGroupRenderer: () => JSX.Element
+  options: object[]
+  valueArray: any
+  valueKey: any
+}
+
+export type DropdownProps = {
+  value?: Value,
+  valueLink?: {
+    value?: Value,
+    requestChange: (value: Value) => void
+  },
+  onChange?: (value?: Value) => void,
```

Comment:
`@types/react-select` defines it as `value?: Option | Option[] | string | string[] | number | number[] | boolean;`